mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/119 checkbox UI lookandfeel

Closes #119 
updated the checkbox ui toggle to match the ui mockups

To test:

- start the game and notice the checkbox on the main menu and in the preference menu
